lib/api/contact-details/v2/types.ts

```diff
@@ -45,9 +45,18 @@ export enum ContactInformationContactTypes {
 }
 
 export enum ContactDetailsPhoneTypes {
-  MOBILE = "mobile",
-  HOME = "home",
-  WORK = "work",
+  MAIN_NUMBER = "mainNumber",
+  EMERGENCY_CONTACT = "emergencyContact",
+  CARER = "carer",
+  WIFE = "wife",
+  HUSBAND = "husband",
+  SPOUSE = "spouse",
+  CHILD = "child",
+  SIBLING = "sibling",
+  RELATIVE = "relative",
+  NEIGHBOUR = "neighbour",
+  DOCTOR = "doctor",
+  SOCIAL_WORKER = "socialWorker",
```

Comment:
How is this text displayed on the frontend? Should it be sentence case?

Reply:
Good question.
In Personal Details there is a locale file that pretty much "converts" these into a nice format. That'll be also edited in the Personal Details PR
![image](https://github.com/LBHackney-IT/mtfh-frontend-common/assets/70756861/bec9de83-ab5b-4317-b6d5-0e8fc84ab079)